maintainers/maintainer-list.nix

```diff
@@ -2782,6 +2782,11 @@
     github = "2chilled";
     name = "Matthias Herrmann";
   };
+  marius851000 = {
```

Comment:
You wanted to add yourself as a maintainer to openmw?

Reply:
No, that was just an error - from other package I created.